src/lib.rs

```diff
@@ -18,10 +36,29 @@ extern crate libc;
 
 
 /// Returns the number of available CPUs of the current system.
+/// 
+/// This function will get the number of logical cores. Sometimes this is different to the number
```

Comment:
Typo: should be `different from the...`.

Maybe we should rewrite it as "This number might be different from the number of..." instead of "Sometimes this is different..."

Reply:
I think they're both valid English, but I'll change to from.

I've looked this up and apparently "different to" is British English mainly, so "different from" is probably more international: https://en.oxforddictionaries.com/usage/different-from-than-or-to